Makefile

```diff
@@ -130,3 +130,15 @@ ci-pr: test-cross
 .PHONY: clean
 clean:
 	rm -rf bin *.log *-kernel *-cmdline *-state *.img *.iso *.gz *.qcow2 *.vhd *.vmx *.vmdk *.tar *.raw
+
+update-package-tags:
```

Comment:
This mostly is a shell script...I don't think this should be in the toplevel Makefile..maybe it should be a shellscript

Reply:
It was in a document, which meant copy-paste and everything with it. This certainly is a step up. I would take this in, then we can iterate if we want.

Reply:
As said, I'd much rather have this as script. The Makefile currently is about building stuff. We already have scripts to manage tags and such in `scripts` so might as well put this there.

Reply:
You want a script in `scripts/` that then calls this `update-component-sha.sh`? 

> Makefile currently is about building stuff

make is about combining common tasks, so I can do `make foo` and `make bar`. That might be "building stuff", and it might be updating stuff. We have lots of non-"build stuff" targets: `bin` creates a directory; `test-cross` runs tests; `clean` removes artifacts. 

Personally, I think the entire alpine and dependency build process should be simplified into a single script with options. But I am not about to tackle that now; maybe after I have had some distance from the last two weeks of painful building and rebuilding (maybe). In the meantime, I just want to capture what I went through and simplify it with low-hanging fruit so the next person doesn't go through it.